charts/umbrella/values-adopter-data-exchange-stub.yaml

```
@@ -0,0 +1,212 @@
+        nginx.ingress.kubernetes.io/proxy-buffering: "on"
+        nginx.ingress.kubernetes.io/proxy-buffers-number: "20"
+        nginx.ingress.kubernetes.io/use-regex: "true"
+      tls: false
+    # -- uncomment the following for persistence
+    # postgresql:
+    #   primary:
+    #     persistence:
+    #       enabled: true
+
+
+bdrs-server-memory:
+  enabled: true
+  seeding:
+    bpnList:
+      - bpn: "BPNL000000000000"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL000000000000"
+      - bpn: "BPNL00000003B3NX"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B3NX"
+      - bpn: "BPNL00000003CSGV"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CSGV"
+      - bpn: "BPNL00000003B6LU"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B6LU"
+      - bpn: "BPNL00000003AXS3"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AXS3"
+      - bpn: "BPNL00000003AZQP"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AZQP"
+      - bpn: "BPNL00000003AWSS"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AWSS"
+      - bpn: "BPNL00000003AYRE"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AYRE"
+      - bpn: "BPNL00000003AVTH"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AVTH"
+      - bpn: "BPNL00000000BJTL"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000000BJTL"
+      - bpn: "BPNL00000003CML1"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CML1"
+      - bpn: "BPNL00000003B2OM"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B2OM"
+      - bpn: "BPNL00000003B0Q0"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B0Q0"
+      - bpn: "BPNL00000003B5MJ"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003B5MJ"
+      - bpn: "BPNS0000000008ZZ"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNS0000000008ZZ"
+      - bpn: "BPNL00000003CNKC"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003CNKC"
+      - bpn: "BPNS00000008BDFH"
+        did: "did:web:ssi-dim-wallet-stub.tx.test:BPNS00000008BDFH"
+  hostname: &hostname bdrs-server.tx.test
+  server:
+    trustedIssuers:
+      - did:web:ssi-dim-wallet-stub.tx.test:BPNL000000000000
+    ingresses:
+      - enabled: true
+        hostname: *hostname
+        endpoints:
+          - directory
+          - management
+        tls:
+          enabled: false
+
+dataconsumerOne:
+  enabled: true
+  secrets:
+    edc-miw-keycloak-secret: BPNL00000003AZQP
+  tractusx-connector:
+    participant:
+      id: BPNL00000003AZQP
+    iatp:
+      id: did:web:ssi-dim-wallet-stub.tx.test:BPNL00000003AZQP
+      trustedIssuers:
+        - did:web:ssi-dim-wallet-stub.tx.test:BPNL000000000000
+      sts:
+        dim:
+          url: http://ssi-dim-wallet-stub.tx.test/api/sts
+        oauth:
+          token_url: http://ssi-dim-wallet-stub.tx.test/oauth/token
+          client:
+            id: BPNL00000003AZQP
```

it's a bit strange to usage a BPN as clientid, isn't it?
I assume due to the stub it's irrelevant what we put there
```suggestion
            id: dim-clientid-1
```

---

The client ID is necessary in the STUB. The previous issue occurred because it was missing. We use the BPNL to indicate to the STUB which 'Wallet' we are working with, ensuring that the exchange functions correctly.

---

Ah ok, so the STUB requires the client ID and client secret to be the BPN? If so, I'd resolve all comments I made for changing those.

---

I'm not sure about the secret, but the Client ID is necessary.

---

I just checked, and the secrets are not necessary.